crates/libcontainer/src/rootfs/mount.rs

```diff
@@ -14,9 +14,13 @@ use libcgroups::common::CgroupSetup::{Hybrid, Legacy, Unified};
 use libcgroups::common::DEFAULT_CGROUP_ROOT;
 use nix::{errno::Errno, mount::MsFlags};
 use oci_spec::runtime::{Mount as SpecMount, MountBuilder as SpecMountBuilder};
+use openat::Dir;
```

Comment:
Why did you chose opendat crate instead of nix's opendat?
https://docs.rs/nix/latest/nix/dir/struct.Dir.html#method.openat

Reply:
I didn't realize openat was on nix.
Thanks for letting me know.

Reply:
fixed